CMakeLists.txt

```diff
@@ -847,15 +847,15 @@ foreach(lib ${LAPACK_LIBRARIES} ${MPI_Fortran_LIBRARIES})
 endforeach()
 string(REPLACE ";" " " PARPACK_PC_LIBS_PRIVATE "${PARPACK_PC_LIBS_PRIVATE}")
 
-configure_file(pkg-config/arpack.pc.in "${PROJECT_BINARY_DIR}/arpack${LIBSUFFIX}${ITF64SUFFIX}.pc" @ONLY)
-configure_file(pkg-config/parpack.pc.in "${PROJECT_BINARY_DIR}/parpack${LIBSUFFIX}${ITF64SUFFIX}.pc" @ONLY)
+configure_file(SRC/arpack.pc.in "${PROJECT_BINARY_DIR}/SRC/arpack${LIBSUFFIX}${ITF64SUFFIX}.pc" @ONLY)
+configure_file(PARPACK/SRC/MPI/parpack.pc.in "${PROJECT_BINARY_DIR}/PARPACK/SRC/MPI/parpack${LIBSUFFIX}${ITF64SUFFIX}.pc" @ONLY)
 
```

Comment:
@turboencabulator Why did you move the pc files? (as the repo is maintained-only, I'd prefer minimal changes to avoid breaking everything!)

Reply:
It's documented in the commit message:  Each directory's Makefile is responsible for building/installing files in that directory.  If parpack isn't selected to be built, we skip over the parpack source directory and its .pc file won't be installed, and we don't need extra decisions elsewhere to skip installing that file.

This is also just moving the files back to their original locations (before 93310367025a01db9c582f2c2a264d8f91826626 moved them and caused this problem).